apps/liteprotocoltester/tester_config.nim

```diff
@@ -0,0 +1,146 @@
+import
+  std/[strutils, strformat],
+  stew/results,
+  chronos,
+  regex,
+  confutils,
+  confutils/defs,
+  confutils/std/net,
+  confutils/toml/defs as confTomlDefs,
+  confutils/toml/std/net as confTomlNet,
+  libp2p/crypto/crypto,
+  libp2p/crypto/secp,
+  libp2p/multiaddress,
+  nimcrypto/utils,
+  secp256k1,
+  json
+import
+  ../../waku/common/confutils/envvar/defs as confEnvvarDefs,
+  ../../waku/common/confutils/envvar/std/net as confEnvvarNet,
+  ../../waku/common/logging,
+  ../../waku/factory/external_config,
+  ../../waku/waku_core
+
+export confTomlDefs, confTomlNet, confEnvvarDefs, confEnvvarNet
+
+const
+  LitePubsubTopic* = PubsubTopic("/waku/2/default-waku/proto")
+  LiteContentTopic* = ContentTopic("/tester/1/light-pubsub-example/proto")
+
+type TesterFunctionality* = enum
+  sender # pumps messages to the network
+  receiver # gather and analyze messages from the network
+
+type LiteProtocolTesterConf* = object
+  configFile* {.
+    desc:
+      "Loads configuration from a TOML file (cmd-line parameters take precedence) for the light waku node",
+    name: "config-file"
+  .}: Option[InputFile]
+
+  ## Log configuration
+  logLevel* {.
+    desc:
+      "Sets the log level for process. Supported levels: TRACE, DEBUG, INFO, NOTICE, WARN, ERROR or FATAL",
+    defaultValue: logging.LogLevel.DEBUG,
+    name: "log-level"
+  .}: logging.LogLevel
+
+  logFormat* {.
+    desc:
+      "Specifies what kind of logs should be written to stdout. Suported formats: TEXT, JSON",
+    defaultValue: logging.LogFormat.TEXT,
+    name: "log-format"
+  .}: logging.LogFormat
+
+  ## Test configuration
+  servicenode* {.desc: "Peer multiaddr to of the service node.", name: "service-node".}:
```

Comment:
super nitpick

```suggestion
  servicenode* {.desc: "Peer multiaddr of the service node.", name: "service-node".}:
```

Reply:
:-) Wow that something mean having good eyes!!! Thx.